aries_cloudagent/transport/inbound/http.py

```diff
@@ -17,7 +16,14 @@
 class HttpTransport(BaseInboundTransport):
     """Http Transport class."""
 
-    def __init__(self, host: str, port: int, create_session, **kwargs) -> None:
+    def __init__(
+        self,
+        host: str,
+        port: int,
+        create_session,
+        root_profile: Profile = None,
```

Comment:
We can probably just let this be a part of the `**kwargs` for the transports not requiring any access to the root_profile.

Reply:
ahh, gotcha.